README.md

```diff
@@ -49,3 +49,49 @@ SNS data workflow manager and reporting app
 	use them automatically.
 
 [![DOI Badge](https://zenodo.org/badge/4139/neutrons/data_workflow.png)](http://dx.doi.org/10.5281/zenodo.10054)
+
+## Setup Dev Environment with docker-compose
+
+During the modernization of WebMon, the developers can setup a local development environment using a database dump from the production database.
+Here are the recommended steps to setup your dev environment:
+
+- Copy the database dump, `dump.sql` to your local system. (asking on Slack chanel for the location of the file)
+- Clone the repository as usual.
+- Install `docker` and `docker-compose` if they are not present on your system.
+  - On Ubuntu, `docker-compose` is not shipped with the system `docker` package.
+  - Grab the pre-build binary for `docker-compose` on the official Github repo and put it in a location in your `$PATH` (such as `/usr/local/bin` or `$HOME/bin`)
+- Cleanup containers, volumes and images from previous run (skip this step if this is the first time)
+  - Use `docker container prune` to prune all stopped containers
+  - Use `docker volume prune` to prune all volumes
+  - [Optional] Use `docker image prune` to remove all images
+- Modify the default web cache name to avoid conflicts with `dump.sql`
+  - In `src/reporting/reporting_app/settings.py` , change L210 into `'LOCATION': 'webcache_dev'`
+  - In `src/Makefile`, change L91 into `cd $(prefix)/app; python manage.py createcachetable webcache_dev`
+- Move to the directory that contains `docker-compose.yml` (the root of repo)
+- Spin up the database container in the background with
+  > `docker-compose up db -d`
+- Import `dump.sql` to the database container with 
+  > `docker exec -i data_workflow-db-1 /bin/bash -c "PGPASSWORD=workflow psql --username workflow workflow" < dump.sql`
```

Comment:
Wrong name here. It is `monitor-data-workflow_db_1`

Reply:
The container name should be correct (see the last column in the image below)
![image](https://user-images.githubusercontent.com/5064852/146828797-555fbd80-a744-4f1f-800a-dfdeb8fb6c98.png)